src/picker/VuePersianDatetimePicker.vue

```diff
@@ -434,6 +453,16 @@ export default {
     event: 'input'
   },
   props: {
+    /**
+     * Next and previous day shortcut
+     * @type Boolean
+     * @default False
+     * @if false, the picker will not show shortcuts
+     * @if true, the picker will show shortcuts
+     * @example <... :day-change-shortcut="true">
+     */
+    dayChangeShortcut: { type: Boolean, default: false },
+
```

Comment:
Please move these lines to end of `props` (Usually new props is added to the end)